### PR TITLE
Fix input events in select and multiselect

### PR DESCRIPTION
**Pull request by micheal-parks**
Fixes instances of both native and custom input events firing, causing state issues or errors to components that listen to these events due to the `detail` object only existing on custom events.

Note: this fixes camera issues we're seeing in RC.

### CODE REVIEW

**Comment by ehhong** [comment]
I don't entirely understand the bug -- was this able to be locally tested with app?

**Comment by micheal-parks** [comment]
@ehhong The issue is that v-select and v-multiselect have hidden checkbox inputs that represent if an item within the selection list has been selected. These hidden inputs will fire `input` events, which will bubble up the DOM to whatever is listening for the `input` custom event that the these elements also fire. There's no reason to listen for these checkbox events so they're silenced by calling `stopPropagation`.

**Comment by ehhong** [comment]
@micheal-parks why would this bug have prevented the manual refresh button from handling that event properly? because there was an exception thrown by these bubbled `input` events?

**Comment by micheal-parks** [comment]
There are unrelated issues with full width buttons that are breaking checks. Not sure how they were merged but working on fixes.

**Comment by ehhong** [comment]
> @ehhong The exception was thrown because a component that was listening to `input` was receiving events that did not contain the `details` object, which it was reading from for event data. `details` is only a feature of the `CustomEvent` class, which is what we're instantiating when firing our custom `input` event, and not the `Event` class, which is firing in native `input` events unless `stopPropagation` is called.

got it! thanks so much for the explanation

**Review by ehhong — APPROVED** [review]

**Review by mrloureed — APPROVED** [review]
LGTM

File: packages/core/src/lib/button.svelte [hunk 1]
```diff
@@ -89,10 +90,3 @@ export let width: 'full' | 'default' = 'default';
     </span>
   {/if}
 </button>
-
-<style>
-:host {
-  --button-display: {display};
```
Comment (micheal-parks) [inline packages/core/src/lib/button.svelte:95]: This isn't valid svelte syntax and was breaking formatting in CI. Not sure how it passed ever. Anyway, there's a much simpler solution that doesn't involve setting styles on the host, and hosts don't even exist in a non-web-component context. Fix works in both core and legacy.

**Pull request merged**